.github/workflows/ci.yaml

```diff
@@ -12,13 +12,20 @@ jobs:
     strategy:
       matrix:
         ruby-version:
-        - '3.0'
         - '3.1'
+        - '3.2'
+        - '3.3'
         rails-version:
         - '7.0'
         - '7.1'
         - '7.2'
+        include:
+        - ruby-version: '3.0'
+          rails-version: '7.0'
+        - ruby-version: '3.0'
+          rails-version: '7.1'
```

Comment:
I merged, but I think an exclude is better - I'll send up a PR

Reply:
I like to be inclusive 🤣